ImputationPipeline/ScoringPart.wdl

```diff
@@ -27,14 +28,20 @@ workflow ScoringImputedDataset {
     Boolean redoPCA = false
   }
 
+  call ExtractIDs as ExtractIDsPopulation {
+  	input:
+  		vcf = population_vcf,
+  		output_basename = population_basename
+  }
   
   call ScoreVcf as ScoreImputedArray {
   	input:
   	vcf = imputed_array_vcf,
   	basename = basename,
   	weights = weights,
   	base_mem = scoring_mem,
-  	extra_args = columns_for_scoring
+  	extra_args = columns_for_scoring,
+  	sites = ExtractIDsPopulation.ids
```

Comment:
This was needed from #30.  Otherwise if there are sites in the "imputed" array which are not in the population array the scoring adjustment will not work correctly.